UT4MasterServer/Controllers/AccountController.cs

```diff
@@ -250,6 +250,7 @@ public async Task<IActionResult> RegisterAccount([FromForm] string username, [Fr
 			return Conflict("You have entered an invalid username");
 		}
 
+		email = email.ToLower();

```

Comment:
```suggestion
		email = email?.ToLower();

```

Reply:
asp.net core makes sure the value is not null. if it is, it will not call the method as far as i know.